### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by srishti-maurya**
Added products listing page
refactored home page
added images

Preview Home Page : https://deploy-preview-2--plan-it-shop-it.netlify.app/
Preview Products Page: https://deploy-preview-2--plan-it-shop-it.netlify.app/pages/products.html

### CODE REVIEW

File: styles/products.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  .products-card {
+    height: 400px !important;
+  }
+}
+
+@media (max-width: 550px) {
+  .products-card {
+    height: 300px !important;
+  }
+}
```
Comment (BelideNikhil) [inline styles/products.css:64]: Try not to use !important

File: styles/products.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+@media (max-width: 500px) {
+  .products-list {
+    grid-template-columns: repeat(1, 1fr);
```
Comment (BelideNikhil) [inline styles/products.css:68]: You could just do 1fr

File: styles/styles.css [hunk 3]
```diff
@@ -1,130 +1,160 @@
...
+  .choice-container {
+    grid-template-columns: repeat(1, 1fr);
   }
-  .card-price {
-    display: none;
+  .genre-container {
+    grid-template-columns: repeat(1, 1fr);
+  }
+  .category {
+    grid-template-columns: repeat(1, 1fr);
```
Comment (BelideNikhil) [inline styles/styles.css:152]: you can just do, 

.choice-container , 
.genre-container , 
.category {
    grid-template-columns: 1fr;
  }

**Review by BelideNikhil — APPROVED** [review]
Great UI.
Keep Going!

**Review by Amandeep-sinha — APPROVED** [review]
You can use .webp format for images

**Comment by Swapnilkota108** [comment]
UI is really good 👍

File: styles/products.css (hunk 2 above)
Comment (srishti-maurya) [reply styles/products.css:68]: Thank you for the review I'll update it.

File: styles/styles.css (hunk 3 above)
Comment (srishti-maurya) [reply styles/styles.css:152]: Thank you for the review I'll update it.

**Pull request merged**